index.html

```diff
@@ -5,9 +5,12 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <script src="./index.js"></script>
+  <script src="./cycles.js"></script>
+  <script src="./functions.js" defer></script>
```

Comment:
if you are working with DOM, where should you insert the script?? and why. answer in DM

Reply:
answered